scripts/globals/mobskills/Deadeye.lua

```diff
@@ -0,0 +1,39 @@
+---------------------------------------------
+--  Deadeye
+--  Family: Qiqurn
+--  Description: Lowers the defense and magical defense of enemies within range.
+--  Type: Magical
+--  Utsusemi/Blink absorb: Ignores shadows
+--  Range: Unknown
+--  Notes: Used only by certain Notorious Monsters. Strong effect.
+---------------------------------------------
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+    return 0;
+end;
+
+function onMobWeaponSkill(target, mob, skill)
+
+    local defDown = false;
+    local mDefDown = false;
+
+    defDown = MobStatusEffectMove(mob, target, EFFECT_DEFENSE_DOWN, 50, 0, 120);
+    mDefDown = MobStatusEffectMove(mob, target, EFFECT_MAGIC_DEF_DOWN, 50, 0, 120);
+
+    skill:setMsg(MSG_ENFEEB_IS);
+
+    -- display defense down first, else magic defense down
+    if (defDown == MSG_ENFEEB_IS) then
+        typeEffect = EFFECT_DEFENSE_DOWN;
+    elseif (mDefDown == MSG_ENFEEB_IS) then

```

Comment:
there are none that show multiple messages (it's not even possible i don't
think)

On Sun, Aug 2, 2015 at 3:23 PM, TeoTwawki <notifications@github.com> wrote:

> In scripts/globals/mobskills/Deadeye.lua
> <https://github.com/DarkstarProject/darkstar/pull/1851#discussion_r36048152>
> :
>
> > +end;
> > +
> > +function onMobWeaponSkill(target, mob, skill)
> > +
> > +    local defDown = false;
> > +    local mDefDown = false;
> > +
> > +    defDown = MobStatusEffectMove(mob, target, EFFECT_DEFENSE_DOWN, 50, 0, 120);
> > +    mDefDown = MobStatusEffectMove(mob, target, EFFECT_MAGIC_DEF_DOWN, 50, 0, 120);
> > +
> > +    skill:setMsg(MSG_ENFEEB_IS);
> > +
> > +    -- display defense down first, else magic defense down
> > +    if (defDown == MSG_ENFEEB_IS) then
> > +        typeEffect = EFFECT_DEFENSE_DOWN;
> > +    elseif (mDefDown == MSG_ENFEEB_IS) then
>
> There's a couple on retail that show multiple messages, but most moves
> only do 1. And several that are "damage and this status" only show the
> damage part, leaving players to later notice they have this here bad effect
> and go "where'd that come from?!" [image: :laughing:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DarkstarProject/darkstar/pull/1851/files#r36048152>.
>